frontend/src/component/changeRequest/ChangeRequest/SegmentTooltipLink.tsx

```diff
@@ -56,9 +56,15 @@ const StyledContainer: FC<{ children?: React.ReactNode }> = styled('div')(
     }),
 );
 
+const ViewDiff = styled('span')(({ theme }) => ({
+    color: theme.palette.primary.main,
+    marginLeft: theme.spacing(1),
+}));
+
 const Truncated = styled('div')(() => ({
     ...textTruncated,
     maxWidth: 500,
+    display: 'flex',
```

Comment:
Out of curiosity, what's the flex doing here? Is it just aligning the items on a row (which used to not be necessary because there was only one item)?

Reply:
yes, preventing elements from being put one below the other. We prefer a horizontal row similar to strategy changes